traefik-mixin/alerts/alerts.libsonnet

```diff
@@ -0,0 +1,33 @@
+{
+  groups+: [
+    {
+      name: 'traefik',
+      rules: [
+        // TraefikConfigReloadFailuresIncreasing
+        {
+          alert: 'TraefikConfigReloadFailuresIncreasing',
+          expr: "sum(rate(traefik_config_reloads_failure_total[5m])) > 0",
+          'for': '5m',
+          labels: {
+            severity: 'critical',
+          },
+          annotations: {
+            description: 'Traefik is failing to reload its config',
+          },
+        },
+        // TraefikTLSCertificatesExpiring
+        {
+          alert: 'TraefikTLSCertificatesExpiring',
+          expr: "max by (sans) ((last_over_time(traefik_tls_certs_not_after[5m]) - time()) / 86400) < 7",
```

Comment:
Would be great to have 7 configurable in config.libsonnet

Reply:
configurable now! 932c72cce9949522402b78e7813af36aa8d62ea3